.gitignore

```diff
@@ -0,0 +1 @@
+.idea
```

Comment:
Can you please keep this in your global gitignore?

https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer